newrelic/resource_newrelic_synthetics_monitor_test.go

```diff
@@ -30,7 +30,7 @@ func TestAccNewRelicSyntheticsBrowserMonitor_DeviceEmulationError(t *testing.T)
 			// Test: Create
 			{
 				Config:      testAccNewRelicSyntheticsBrowserMonitorConfig_DeviceEmulationError(rName, string(SyntheticsMonitorTypes.BROWSER)),
-				ExpectError: regexp.MustCompile(`you need to specify both 'device_type' and 'device_orientation' fields`),
+				ExpectError: regexp.MustCompile("all of `device_orientation,device_type` must be\nspecified"),
```

Comment:
is the `\n` intended 🤔

Reply:
yes, this is intended, due to the error message format is quite weird as shown below
<img width="875" alt="Screenshot 2024-09-11 at 11 16 00 AM" src="https://github.com/user-attachments/assets/4f216d95-9081-4bb2-b129-fb0bd3a8d557">